extensions/TheShovel/ShovelUtilsTW.js

```diff
@@ -0,0 +1,257 @@
+console.log("ShovelUtils v1.1")
+tempImageLoad = null
+string = null;
```

Comment:
```suggestion
var string = '';
```
fixes string is not defined
also unused